src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/format/EncodingService.kt

```diff
@@ -1,28 +1,24 @@
-package uk.gov.justice.digital.hmpps.personrecord.message.processors.probation
+package uk.gov.justice.digital.hmpps.personrecord.service.format
 
 import kotlinx.coroutines.runBlocking
 import org.slf4j.LoggerFactory
-import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.beans.factory.annotation.Value
-import org.springframework.stereotype.Component
+import org.springframework.stereotype.Service
 import uk.gov.justice.digital.hmpps.personrecord.client.CorePersonRecordAndDeliusClient
+import uk.gov.justice.digital.hmpps.personrecord.client.PrisonerSearchClient
 import uk.gov.justice.digital.hmpps.personrecord.client.model.offender.ProbationCase
+import uk.gov.justice.digital.hmpps.personrecord.client.model.prisoner.Prisoner
 import uk.gov.justice.digital.hmpps.personrecord.service.RetryExecutor
 
-@Component
-class BaseProbationEventProcessor {
-
-  @Autowired
-  private lateinit var corePersonRecordAndDeliusClient: CorePersonRecordAndDeliusClient
+@Service
```

Comment:
I know it doesn't matter, but this annotation does nothing that @Component doesn't do. It feels like some idea that someone had about 20 years ago that got as far as the language spec and no further :-)